MediaBrowser.Api/Sessions/SessionService.cs

```diff
@@ -24,10 +21,10 @@ namespace MediaBrowser.Api.Session
     [Authenticated]
     public class GetSessions : IReturn<SessionInfo[]>
     {
-        [ApiMember(Name = "ControllableByUserId", Description = "Optional. Filter by sessions that a given user is allowed to remote control.", IsRequired = false, DataType = "string", ParameterType = "query", Verb = "GET")]
```

Comment:
why remove `Optional`?

Reply:
Optional is implied with `IsRequired` set to false, which should be clear on its own.